build/DownloadSonarPlugin/Common.cs

```diff
@@ -127,34 +127,6 @@ public static void UnzipJar(string jarFilePath, string destinationFolder, TaskLo
             LogElapsedTime("Unzipped jar file", timer, logger);
         }
 
-        public static void UncompressAndUnzipTgx(string tarFilePath, string destinationFolder, TaskLoggingHelper logger)
```

Comment:
These methods were only used when processing the CFamily plugin, the processing of which is now handled by the `CFamilyJarPreProcessor` tool.